spec/core_ext/array/tableize_spec.rb

```diff
@@ -34,6 +34,17 @@
         expect(test.tableize).to eq(expected)
       end
 
+      it 'removes color from length' do
+        test = [["Col1", "Col2"], ["Val1", "Val2"], ["\033[31mVal\e[0m", "Value4"]]
+        expected = <<-EOF
+ Col1 | Col2
+------|--------
+ Val1 | Val2
+ Val  | Value4
```

Comment:
So this is probably going to fail currently since if you are keeping the color code in the result, it will be "width'd" to the length of the raw string, but still display the color code in the output here.

Reply:
Also, seems like your regexp still needs some work, since it seems to be deleting `"Val"` from the output:

https://travis-ci.com/github/ManageIQ/more_core_extensions/jobs/359385835#L278

Reply:
```diff
-       expected: " Col1 | Col2\n------|--------\n Val1 | Val2\n Val  | Value4\n"
+            got: " Col1 | Col2\n------|--------\n Val1 | Val2\n \e[31 | Value4\n"
```

I was actually expecting:

```
            got: " Col1 | Col2\n------|--------\n Val1 | Val2\n \e[31mVal\e[0m | Value4\n"
```